packages/prosekit/CHANGELOG.md

```
@@ -1,5 +1,138 @@
 # prosekit
 
+## 0.22.0
+
+### Minor Changes
+
+- [`ea024fe`](https://github.com/ocavue/prosekit/commit/ea024fe8721eb838f074dc8a9a75bccfe968b5a3) ![](https://prosekit.dev/b/extensions) ![](https://prosekit.dev/b/web) ![](https://prosekit.dev/b/react) ![](https://prosekit.dev/b/preact) ![](https://prosekit.dev/b/solid) ![](https://prosekit.dev/b/vue) ![](https://prosekit.dev/b/svelte)
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)